docs/content/getting-started/perform-list-users.mdx

```diff
@@ -0,0 +1,190 @@
+3. You have loaded `FGA_STORE_ID` and `FGA_API_URL` as environment variables.
+
+</TabItem>
+</Tabs>
+
+## Step By Step
+
+Assume that you want to list all objects of type document that  user `anne` has `reader` relationship with:
+
+### 01. Configure the <ProductName format={ProductNameFormat.ShortForm}/> API Client
+
+Before calling the check API, you will need to configure the API client.
+
+<Tabs groupId="languages">
+<TabItem value={SupportedLanguage.JS_SDK} label={languageLabelMap.get(SupportedLanguage.JS_SDK)}>
+
+<SdkSetupHeader lang={SupportedLanguage.JS_SDK} />
+
+</TabItem>
+<TabItem value={SupportedLanguage.GO_SDK} label={languageLabelMap.get(SupportedLanguage.GO_SDK)}>
+
+<SdkSetupHeader lang={SupportedLanguage.GO_SDK} />
+
+</TabItem>
+<TabItem value={SupportedLanguage.DOTNET_SDK} label={languageLabelMap.get(SupportedLanguage.DOTNET_SDK)}>
+
+<SdkSetupHeader lang={SupportedLanguage.DOTNET_SDK} />
+
+</TabItem>
+<TabItem value={SupportedLanguage.PYTHON_SDK} label={languageLabelMap.get(SupportedLanguage.PYTHON_SDK)}>
+
+<SdkSetupHeader lang={SupportedLanguage.PYTHON_SDK} />
+
+</TabItem>
+<TabItem value={SupportedLanguage.JAVA_SDK} label={languageLabelMap.get(SupportedLanguage.JAVA_SDK)}>
+
+<SdkSetupHeader lang={SupportedLanguage.JAVA_SDK} />
+
+</TabItem>
+<TabItem value={SupportedLanguage.CLI} label={languageLabelMap.get(SupportedLanguage.CLI)}>
+
+<SdkSetupHeader lang={SupportedLanguage.CLI} />
+
+</TabItem>
+<TabItem value={SupportedLanguage.CURL} label={languageLabelMap.get(SupportedLanguage.CURL)}>
+
+To obtain the [access token](https://auth0.com/docs/get-started/authentication-and-authorization-flow/call-your-api-using-the-client-credentials-flow):
+
+<SdkSetupHeader lang={SupportedLanguage.CURL} />
+
+</TabItem>
+</Tabs>
+
+### 02. Calling List Objects API
+
+To return all documents that user `user:anne` has relationship `reader` with:
+
+<ListUsersRequestViewer
+  authorizationModelId="01HVMMBCMGZNT3SED4Z17ECXCA"
+  objectType="document"
+  objectId="planning"
+  relation="reader"
+  userFilterType="user"
+  expectedResults={{
+    users: [{ object: { type: "user:anne" } }],
+    excluded_users: [{ object: { type: "user:beth" } }]
+  }}
+  skipSetup={true}
+  allowedLanguages={[
+    SupportedLanguage.JS_SDK,
+    SupportedLanguage.GO_SDK,
+    SupportedLanguage.DOTNET_SDK,
+    SupportedLanguage.PYTHON_SDK,
+    SupportedLanguage.JAVA_SDK,
+    SupportedLanguage.CLI,
+    SupportedLanguage.CURL,
+  ]}
+/>
+
+The result `document:otherdoc` and `document:planning` are the document objects that `user:anne` has `reader` relationship with.
+
+:::caution Warning
+The performance characteristics of the ListUsers endpoint vary drastically depending on the model complexity, number of tuples, and the relations it needs to evaluate. Relations with 'and' or 'but not' are more expensive to evaluate than relations with 'or'.
+:::
```

Comment:
We have hunches about performance considerations but not enough to make assertions like this IMO. I would remove.

Reply:
Not a problem! Added in 0ba8391bc960ba565d1e0f7e5fa6ab1fae8460bd